pa11y/mappings/work/data/covers/covers.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "id": "870970-basis:134693959",
+    "type": "pid",
+    "imageUrls": {
+      "small": {
+        "url": "https://res.cloudinary.com/dandigbib/image/upload/t_ddb_cover_small/v1647612411/bogportalen.dk/9788702307566.jpg",
+        "format": "jpeg",
+        "size": "small"
```

Comment:
Consider if this file is still relevant.

As far as I can tell it has been superseeded by a mock returning images from picsum.

Reply:
Removed